### PR TITLE
completes get /api/users and tests

### PR DESCRIPTION
**Pull request by caffreydev**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -537,3 +537,44 @@ describe('delete /api/comments/:comment_id', () => {
...
+      .then(({ body }) => {
+        expect(body.users.length).toBe(4);
+        body.users.forEach((user) => {
+          expect(user).toMatchObject({
+            username: expect.any(String),
+            name: expect.any(String),
+            avatar_url: expect.any(String),
+          });
+          expect(Object.keys(user).length).toBe(3);
+        });
```
Comment (Jodiech) [inline __tests__/app.test.js:568]: Line 567 can be deleted as you are already checking this with the **toMatchObject** above 👍

File: __tests__/app.test.js [hunk 2]
```diff
@@ -537,3 +537,44 @@ describe('delete /api/comments/:comment_id', () => {
...
+        expect(Array.isArray(body.users)).toBe(true);
+      });
+  });
+
+  it('response array should be composed of objects of the right format', () => {
+    return request(app)
+      .get('/api/users')
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.users.length).toBe(4);
```
Comment (Jodiech) [inline __tests__/app.test.js:560]: Nice that you are checking the length here 😄

**Review by Jodiech — APPROVED** [review]
Hey 👋 nice work! Your code is great! Happy for you to merge 🚀

File: __tests__/app.test.js (hunk 1 above)
Comment (caffreydev) [reply __tests__/app.test.js:568]: only there as matchobject will ignore extra keys that have ended up in there, so may as well test for it not picking up some rogue key

**Pull request merged**